docs/recipes/WritingTests.md

```diff
@@ -8,12 +8,39 @@ hide_title: true
 
 # Writing Tests
 
```

Comment:
:hand: If we're updating this page, let's add a "What You'll Learn" `tip` block here similar to what's in the tutorial pages

Reply:
Yup, added.

I couldn't quite think of anything to fill it with besides "Recommended practices for testing apps using Redux"